selftests/functional/test_output.py

```diff
@@ -296,12 +296,12 @@ def test_check_on_off(self):
             self.assertTrue(os.path.exists(stdout_path))
             with open(stdout_path, 'r') as stdout:
                 self.assertEqual(stdout.read(),
-                                 '__STDOUT_CONTENT____STDOUT_DO_RECORD_CONTENT__')
+                                 '__STDOUT_CONTENT__\n__STDOUT_DO_RECORD_CONTENT__\n')
```

Comment:
@beraldoleal I've doubled checked and the test does not write new lines, see: https://github.com/avocado-framework/avocado/blob/da13e14b0e42ad364bfb68b3a43cb3b637fa32aa/selftests/functional/test_output.py#L110

The question is now how/if the logging handler used to generate the `stdout` (and similar) files can receive the verbatim content (including newlines) to pass it along.  I'm still investigating that.